cfme/fixtures/service_fixtures.py

```diff
@@ -173,15 +177,17 @@ def order_service(appliance, provider, provisioning, dialog, catalog, request):
     assert provision_request.is_succeeded()
     if provision_request.exists():
         provision_request.wait_for_request()
-        if not BZ(1646333, forced_streams=['5.10']).blocks:
+        if not BZ(1775779, forced_streams=['5.10', '5.11']).blocks:
```

Comment:
:+1:

Reply:
Don't we want to actually work around this BZ, not just skip the remove_request call?

If I understand the BZ correctly, we can still remove, we just need to navigate away first?

Reply:
I am actually proposing this change in the fixture: 
https://github.com/ManageIQ/integration_tests/pull/9663/files#diff-c43bfe6be31a2fdeb6a2768d21c9afe5R175

We don't care if UI/API is used, all we care is to remove the request. Hence using API method. And it works well, I have tested it.

Reply:
Yes, that works .